src/Application.vala

```diff
@@ -0,0 +1,202 @@
+using Gtk;
+
+namespace Ilia {
+    class Application : Gtk.Application {
```

Comment:
Adding this is something I had always planned to do but never got to.  Nice work!

Reply:
Hehe :)